### PR TITLE
Add fn.zeros, fn.zeros_like, fn.ones, fn.ones_like, fn.full and fn.full_like

### PR DESCRIPTION
**Pull request by jantonguirao**
## Category:

**New feature**

## Description:

Creates a set of operators analogous to np.zeros, np.zeros_like, np.ones, np.ones_like, np.full, and np.full_like

### Affected modules and functionalities:

New ops

### Tests:

- [ ] Existing tests apply
- [x] New tests added
  - [x] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [x] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [x] Implements new requirements
- [ ] Affects existing requirements
- [ ] N/A

**REQ IDs**: FULL.[01-04]

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

File: dali/operators/generic/constant_value.cc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      tp.RunAll();
+    ), (  // NOLINT
+      DALI_FAIL(make_string("Data type ", dtype, " is currently not supported. "
+                            "Supported types are : ", ListTypeNames<DALI_CONSTANT_VALUE_TYPES>()));
+    ));  // NOLINT
+  }
+}
+
+DALI_SCHEMA(Full)
+    .DocStr(R"code(Returns new data of given shape and type, filled with a `fill_value`.)code")
```
Comment (JanuszL) [inline dali/operators/generic/constant_value.cc:72]: `fill_value` is not named argument. Can you update wording?

Comment (jantonguirao) [reply dali/operators/generic/constant_value.cc:72]: done

File: qa/TL1_python-self-test-slow/test.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 # used pip packages
-pip_packages='${python_test_runner_package} dataclasses numpy opencv-python pillow librosa==0.8.1 scipy nvidia-ml-py==11.450.51 numba lz4'
+pip_packages='${python_test_runner_package} dataclasses numpy opencv-python pillow librosa scipy nvidia-ml-py==11.450.51 numba lz4'
```
Comment (JanuszL) [inline qa/TL1_python-self-test-slow/test.sh:3]: Is it safe? What about our tests? See https://github.com/librosa/librosa/issues/1445#issuecomment-1033783731.

Comment (jantonguirao) [reply qa/TL1_python-self-test-slow/test.sh]: Moved to a separate PR

**Review by banasraf — APPROVED** [review]

**Review by awolant — APPROVED** [review]

**Pull request merged**